lib/alephant/lookup/lookup_cache.rb

```diff
@@ -26,11 +29,20 @@ def get(key, &block)
         begin
           versioned_key = versioned key
           result = @client.get versioned_key
-          logger.info "Alephant::LookupCache#get key: #{versioned_key} - #{result ? 'hit' : 'miss'}"
-          logger.metric "GetKeyMiss" unless result
+          logger.info(
+            event:  'Alephant::LookupCache#get',
+            key:    versioned_key,
+            result: result ? 'hit' : 'miss'
+          )
+          logger.metric 'GetKeyMiss' unless result
           result ? result : set(key, block.call)
-        rescue StandardError => e
-          block.call if block_given?
+        rescue StandardError => error
+          logger.error(
+            event:  'LookupCacheError',
+            method: "#{self.class}#get",
+            error:  error
+          )
+          block.call
```

Comment:
Just wanted to be clear that this gets rid of the `block_given?` conditional but I think that's ok given that there's no real point in this method if it doesn't provide a block.